### ISSUE TITLE
No sound in video

### ISSUE THREAD

**Issue by Twilz201** [OpenShot/openshot-qt#1599]
Creating a video for school and when I came home from school to work on it, no sound. Last night it was fine. Any help would be great

**Comment by N3WWN** [comment]
@Twilz201 - Can you give this a try:  `Export Video` -> `Advanced` -> `Audio Settings` -> `Audio codec` and change `ac3` to `aac` : 

<img width="316" alt="image" src="https://user-images.githubusercontent.com/19431804/40197635-0d7b4bce-59e2-11e8-912f-d5c557168a38.png">

...now export your video and see if the audio is in tact.

**Comment by Twilz201** [comment]
That worked thank you!  Another issue is that when I am previewing the video its in slow motion is this a computer issue or a setting I can disable?

**Comment by N3WWN** [comment]
@peanutbutterandcrackers - I've noticed the same thing with recent releases of OpenShot.  I just created a new project, set it to 360p @ 30 fps (`Mobile 360p` profile), set the caching preferences to Memory with 250MB cache limit, shrank the preview window to "minimum" (about the width of the play controls), put a single still image on the timeline and if I hit `Play`, the preview only plays at ~1 frame per second.  

It is not an issue with system performance, IO bottlenecks, RAM, number of cores, etc, because, when playing back the preview:
 - System load is 0.2-0.3
 - All 8 cores are mostly idle (97-100% idle on every core)
 - iowait is 0
 - I have over 4 GB of free RAM and no swap in use
 - The content is cached (blue segment on top of the timeline)

It doesn't appear to be a caching problem because playback is still slow if I move the playhead back a few seconds and play only cached content.

I mentioned this on our [Slack](https://openshot.slack.com/archives/C0SULJM36/p1523567628000109) and didn't get any feedback.

@Twilz201 - Is your preview playing at approx 1 frame per second, too?  Is it like this even when the blue line on the top of the timeline extends beyond the playhead location (meaning that the frames about to be displayed are pre-cached)?
